<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Midas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Candara</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:400;&quot;&gt;Midas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnSelectivity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/Selectivity.png</normaloff>icons/Selectivity.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRedox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/RedoxPotential.png</normaloff>icons/RedoxPotential.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSIExporter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/SI_Convertor.png</normaloff>icons/SI_Convertor.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEProfile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/Chemdraw48.png</normaloff>icons/Chemdraw48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnG09">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>G09</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTDPlot">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>TD</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
